tests/Aspire.Hosting.Tests/Dashboard/Playwright/AppbarTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Helpers;
+using Microsoft.Playwright;
+using Xunit;
+
+namespace Aspire.Hosting.Tests.Dashboard;
+
+public class AppbarTests : IClassFixture<PlaywrightFixture>
+{
+    private readonly PlaywrightFixture _playwrightFixture;
+
+    public AppbarTests(PlaywrightFixture playwrightFixture)
+    {
+        _playwrightFixture = playwrightFixture;
+    }
+
+    [LocalOnlyFact]
+    public async Task Dashboard_Test_Change_Theme()
+    {
+        var (_, page) = await _playwrightFixture.SetupDashboardForPlaywrightAsync();
+        var settingsButton = page.GetByRole(AriaRole.Button, new()
+        {
+            Name = "Launch settings"
```

Comment:
This is matching the English string, right? Is there a way to find this button on any locale, as we want to test all localized builds. Perhaps via a `class` or `id` ?